configs/sabaki/README.md

```diff
@@ -0,0 +1,72 @@
+# How to set up sabaki
+
+You can install Sabaki from https://sabaki.yichuanshen.de/.
+If you are on MacOS, you can install using homebrew with the command
+`brew install sabaki`.
+
+## Bot setup
+0. First build the C++ katago executable.
+   To do this, 
+   you'll want to run the following commands in both the
+   `engines/KataGo-custom/cpp`
+   and
+   `engines/KataGo-raw/cpp`
+   directories.
+   ```
+   cmake . -DUSE_BACKEND=OPENCL -DUSE_TCMALLOC=1
+   make -j
+   ```
+   Adjust the backend and whether to use tcmalloc as needed.
+1. Open up Sabaki preferences by pressing command comma (⌘,).
+2. Go to the Engines tab.
+3. A sample config would be the following:
+  ![bot-config-screenshot](bot-config-screenshot.png)
+  The first line is the bot name,
+  the second line is the path to the executable,
+  the third line are the flags passed to the executable,
+  are initial commands passed to gtp
+  (in this case giving the bot infinite time to make moves).
+
+## List of bot configs
+
+Adversary models can be downloaded from
+https://drive.google.com/drive/folders/1-bGX-NQOh6MuRPoXJgYHb9-jWRJvviSg?usp=sharing.
+Base KataGo models can be downloaded from
+https://katagotraining.org/networks/.
+
```

Comment:
would be nice to describe what each of these configs actually mean, especially how they relate to the adversaries/victims described in the paper. That could wait until a later, more final version of our paper though